pkg/pillar/zboot/zboot.go

```diff
@@ -402,6 +402,9 @@ func MarkCurrentPartitionStateActive() error {
 
 // XXX known pathnames for the version file and the zededa-tools container
 const (
+	newVersionFile = "/hostfs/etc/eve-release"
+	// XXX remove use of shortVersionFile in container once the deployed
```

Comment:
I am thinking that we may be better off avoiding a naming game like this one if we actually mount a version file into /etc/ of the container so that the name becomes /etc/eve-release

What do you think @nordmark ?

Reply:
You mean end up referring to /etc/eve-release instead of /hostfs/etc/eve-release?
As long as we just bind mount the file (and not all of /etc) it should be fine.

Reply:
Yup -- that's what I'm proposing. Should we update the patch to change the name and also update build.yml for bind mounting /etc/eve-release ?

Reply:
Done